onadata/apps/logger/signals.py

```diff
@@ -21,3 +56,41 @@ def pre_delete_attachment(instance, **kwargs):
         attachment.media_file.delete()
     except Exception as e:
         logging.error('Failed to delete attachment: ' + str(e), exc_info=True)
+
+
+@receiver(post_save, sender=Attachment)
+def update_user_profile_attachment_storage_bytes(instance, created, **kwargs):
+    """
+    Update the attachment_storage_bytes field in the UserProfile model
+    when an attachment is added
+    """
+    if not created:
+        return
```

Comment:
should be okay for now because we always create a new `Attachment` object when the file content differs, even when replacing one attachment with another identically-named one:
https://github.com/kobotoolbox/kobocat/blob/22dd243dccd31bbdb8a209435d2ef9c5694d3610/onadata/libs/utils/logger_tools.py#L559-L568